docs/elastic.md

```diff
@@ -50,3 +88,311 @@ spec:
       file_size: 0
       files: 100000
 ```
+
+### Prometheus Data
+To enable this functionality we must provide the following parameters to the workload CR file:
+
+```
+  prometheus:
+    es_server: my.es.server
+    es_port: 8080 
+    prom_url: http://< URL > 
+    prom_token: <token>
+    es_parallel: true|false
+```
+
+Depending on your infrastructure and desire to store prometheues data separately from your benchmark data, you are able 
+to specify the same or different ES server/port from the benchmark data instance. Users are also able to separately 
+control whether or not to upload using the parallel indexer. 
+
+It's important to note due to the quantity of data from Promethues we recommend using parallel, as long as your ES 
+infrastructure can handle the load. 
+
+Example CR with elasticsearch and prometheus information provided
+
+```
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: smallfile-benchmark
+  namespace: my-ripsaw
+spec:
+  test_user: homer_simpson
+  clustername: test_ci
+  elasticsearch:
+    server: my.es.server
+    port: 8080
+    index_name: ripsaw-smallfile
+  prometheus:
+    es_server: my.es.server
+    es_port: 8080 
+    prom_url: http://< URL > 
+    prom_token: <token>
+    es_parallel: true
+  metadata:
+    collection: true
+  workload:
+    name: smallfile
+    args:
+      clients: 2
+      operation: ["create", "read", "append", "delete"]
+      threads: 1
+      file_size: 0
+      files: 100000
+```
+   
+#### Retrieving Openshift Prometheus info
+
+In order to query Prometheus we must provide the URL and access token. 
+
+The URL can be captured by executing `oc -n openshift-monitoring get routes | grep prometheus`. 
```

Comment:
It's also possible (and recommended), to use the internal prometheus service addres, which is always the same for every cluster and prevents traffic to go out and in the cluster. (https://prometheus-k8s.openshift-monitoring.svc.cluster.local:9091)